src/screens/ContactScreen/index.js

```diff
@@ -195,9 +206,14 @@ const ContactScreen = ({navigation}) => {
     <SafeAreaView style={styles.container}>
       <StatusBar translucent={false} />
       <Header
-        title={StringConstant.chatTabHeaderCreateChatButtonText}
+        title={
+          sourceScreen === ANONYMOUS
+            ? StringConstant.chatTabHeaderCreateAnonChatButtonText
+            : StringConstant.chatTabHeaderCreateChatButtonText
+        }
         containerStyle={styles.containerStyle}
         subTitle={'Next'}
+        subtitleStyle={selectedUsers.length > 0 && styles.subtitleStyle(sourceScreen)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The title and subtitle styles of the Header component have been updated to handle different scenarios based on the source screen. If the source screen is `ANONYMOUS`, a specific string constant and color are used; otherwise, different string constants and colors are used. This change enhances the flexibility of the UI customization process.
```diff
+ title={
+   sourceScreen === ANONYMOUS
+     ? StringConstant.chatTabHeaderCreateAnonChatButtonText
+     : StringConstant.chatTabHeaderCreateChatButtonText
+ }
+ containerStyle={styles.containerStyle}
+ subTitle={'Next'}
+ subtitleStyle={selectedUsers.length > 0 && styles.subtitleStyle(sourceScreen)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->